common/src/Model/GameFactory.cpp

```diff
@@ -176,18 +175,15 @@ namespace TrenchBroom {
             loadCompilationConfig(config);
             loadGameEngineConfig(config);
             
-            // sneak in the brush content type for tutorial brushes
-            const auto flag = 1 << config.brushContentTypes().size();
-            config.addBrushContentType(Tutorial::createTutorialBrushContentType(flag));
+            const auto configName = config.name();
+            m_configs.emplace(std::make_pair(configName, std::move(config)));
+            m_names.push_back(configName);
```

Comment:
could wrap the code using `GameConfig config` in an extra `{}` block so no one accidentally accesses it after the std::move

Reply:
I agree, but this makes scoping of the `configName` variable awkward. I'd have to drop the const.